LizarroAnghela/Collections/src/Main.java

```diff
@@ -0,0 +1,18 @@
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class Main {
+
+    public static void main(String[] args) {
+	// write your code here
+        ArrayList<Circles> arrayList = new ArrayList<Circles>();
+        arrayList.add(new Circles(2.5, 5.5));
+        arrayList.add(new Circles(4.5,9.5));
+
+        //Collection.class arrayList;
+
+        for (Circles rd: arrayList){
+            System.out.println(rd);
```

Comment:
When you print an object of a class using the console, the default output will be:
ClassName@Memory address
Which is not human readable.
You have two options in here:
1: Properly format the message you would like to print: System.out.println("Circle with radius:" + rd.getRadius());
2: This is better approach, you will need to override the method toString() that comes by default for all the classes. It will be something like this:
public String toString() {
    return "Circle with radius:" + this.radius + " and diameter:" + this.diameter;
}